composer.json

```diff
@@ -14,7 +14,8 @@
     ],
 
     "require": {
-        "bolt/filesystem": "^2.0",
+        "bolt/filesystem": "^2.1",
+        "contao/imagine-svg": "^0.1.2",
```

Comment:
That supports `>= 0.1.2` and `< 0.2.0`. Or are you saying they'll break BC on a 0.1.x release?